tests/test_logger.py

```diff
@@ -0,0 +1,149 @@
+"""Unit tests for logger module."""
+
+import logging
+import unittest
+from unittest import mock
+
+from PyQt5.QtCore import QObject
+from PyQt5.QtWidgets import QApplication
+
+from iquip.apps import logger
+
+class _SignallerTest(unittest.TestCase):
+    """Unit tests for the _Signaller class."""
+
+    def setUp(self):
+        self.qapp = QApplication([])
+
+    def tearDown(self):
+        del self.qapp
+
+    def test_signal_connection(self):
```

Comment:
I'm really sorry to move this test into `_Signaller` test..

I think this is so obvious that it is unnecessary.